packages/kit/src/core/sync/sync.js

```diff
@@ -72,7 +70,7 @@ export async function all_types(config, mode) {
 }
 
 /**
- * Regenerate server-internal.js in response to src/{app.html,error.html,service-worker.js} changing
+ * Regenerate server/internal.js in response to src/{app.html,error.html,service-worker.js} changing
```

Comment:
Should we use `__SERVER__/internal.js` here instead for better discoverability when doing full text search?

Reply:
good idea, done